docs/api/apiv3/tags/baseline_comparisons.yml

```diff
@@ -0,0 +1,245 @@
+---
+description: |-
+  Baseline comparisons allow to compare work packages or collections of work packages with respect to different points in time.
+
+  This helps to answer questions like:
+
+  - Which work packages match a certain set of filters today, which work packages match this set of filters at a certain earlier point in time?
+  - Which properties of these work packages have changed with respect to these points in time?
+
+  This tool can be used to analyze how a project plan has changed with respect to a certain baseline date.
+
+  ## Retrieve Work Packages for Different Timestamps
+
+  The work-packages API supports a `timestamps` parameter to gather information about a single work package or a collection work packages for several points in time.
+
+  ```
+  GET /api/v3/work_packages?timestamps=2022-01-01T00:00:00Z,PT0S
+
+  GET /api/v3/work_packages/123?timestamps=2022-01-01T00:00:00Z,PT0S
+  ```
+
+  Each timestamp should be given as [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) string, either as absolute date and time with timezone, e.g. `"2022-01-01T00:00:00Z"`, or as relative timestamp utilizing the [ISO-8601-Duration](https://en.wikipedia.org/wiki/ISO_8601#Durations) format, e.g. `"P-1Y"`, which is composed of an initial `"P"` for "Period", and a duration. `"P-1Y"` is interpreted as the relative timestamp "1 year ago".
+
+  Several timestamps should be passed as comma-separated list of these ISO-8601 strings to the `timestamps` parameter.
+
+  The timestamps should be given in ascending temporal order, such that the first timestamp is the **baseline** timestamp, and the last timestamp is the **current** timestamp.
+
+  The information of the work packages at the different `timestamps` is returned in the `"_embedded"` section `"attributesByTimestamp"`.
+
+  ```json
+  // /api/v3/work_packages?timestamps=2022-01-01T00:00:00Z,PT0S
+  {
+    "_type": "WorkPackageCollection",
+    "total": 1,
+    "_embedded": {
+      "elements": [
+        {
+          "_type": "WorkPackage",
+          "id": 1528,
+          "subject": "Current subject of the work package",
+          // other attributes ...,
+          "_links": {
+            "self": {
+              "href": "/api/v3/work_packages/1528?timestamps=2022-01-01T00:00:00Z,PT0S"
+            }
+          },
+          "_meta": {
+            "matchesFilters": true,
+            "exists": true,
+            "timestamp": "PT0S"
```

Comment:
The `timestamp` property is never explicitly explained. It should be rater self explanatory, but it can't hurt to have one sentence regarding it.

Reply:
Previously I had not mentioned the `timestamps` property, because we weren't completely sure that we would keep the `timestamps` property. The information is redundant in the sense that the array order matches the order of the requested timestamps; so we already know the timestamp.

However, I like adding the `timestamp`  property explicitly to have a check and also for clarity.

I've added a short explanation for the `timestamp` property in https://github.com/opf/openproject/pull/11783/commits/5a33ab64d384e89c6da23412661af7e4aea2182e.